cli/parse_pdfs.py

```diff
@@ -16,7 +16,7 @@
 import requests
 from cloudpathlib import CloudPath, S3Path
 from fitz.fitz import EmptyFileError
-from layoutparser import load_pdf, Layout, draw_box
+from layoutparser import load_pdf, Layout, draw_box  # type: ignore
```

Comment:
@kdutia @stefl14 
This type ignore also isn't needed, the following will work:

        from layoutparser.io import load_pdf
        from layoutparser.elements import Layout
        from layoutparser.visualization import draw_box 
        
If you look at the following __init__.py of the package it lists import structure. 

https://github.com/Layout-Parser/layout-parser/blob/main/src/layoutparser/__init__.py